internal/reorder/stackbranch.go

```diff
@@ -0,0 +1,61 @@
+package reorder
+
+import (
+	"github.com/spf13/pflag"
+	"strings"
+)
+
+// StackBranchCmd is a command to create a new branch in a stack.
+//
+//	branch <branch-name> [--parent <parent-branch-name>] [--trunk <trunk-branch-name>]
```

Comment:
stack-branch

Reply:
Yep, thanks. Originally it was named just `branch` (before the decision to make it `stack-branch` to avoid conflicts with rebase's `break`) and I just missed these. Thanks for catching!